### PR TITLE
Translate scale bar & north arrow

### PR DESCRIPTION
**Pull request by hwbllmnn**
@terrestris/devs please review

### CODE REVIEW

File: src/view/form/Print.js [hunk 1]
```diff
@@ -744,8 +744,10 @@ Ext.define('BasiGX.view.form.Print', {
             xtype: 'checkbox',
             name: attributeRec.get('name'),
             checked: true,
-            fieldLabel: attributeRec.get('name'),
-            boxLabel: '…verwenden?' // TODO i18n
+            bind: {
+                fieldLabel: '{' + attributeRec.get('name') + '}',
+                boxLabel: '{labelUse}'
```
Comment (weskamm) [inline src/view/form/Print.js]: where does `labelUse` come from?

Comment (hwbllmnn) [reply src/view/form/Print.js]: In my case that comes from the project specific i18n files. Does it need to be added somewhere in BasiGX, too?

Comment (weskamm) [reply src/view/form/Print.js]: Yes, please add it to the viewmodels data ontop of your class

**Comment by weskamm** [comment]
nice, please merge once travis goes green

**Pull request merged**